generators/app/templates/src/app/components/RadioGroup/components/RadioOption/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { IRadioOption } from './interfaces';
```

Comment:
We are naming the interface Props. We should keep a convention for that.

I'd move the interface here and name it `Props`. What do you think?
@damfinkel @lucasnestrojil

Reply:
I have doubts with this. If i need use this interface out of component. How i have to do?
If i move into `index` i've to change export type. 

`IRadioOption` is used in the render
> {options.map((childProps: IRadioOption[]) => ( ...)

@pabloferro @damfinkel